src/Mailgun/Resource/Api/Suppressions/Bounce/ShowResponse.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Resource\Api\Suppressions\Bounce;
+
+use Mailgun\Resource\ApiResponse;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class ShowResponse implements ApiResponse
+{
+    /**
+     * @var string
+     */
+    private $address;
+
+    /**
+     * @var string
+     */
+    private $code;
+
+    /**
+     * @var string
+     */
+    private $error;
+
+    /**
+     * @var \DateTime
+     */
+    private $created_at;
```

Comment:
Parameter name should be `$createdAt` instead of `$created_at`. Can you update it please? Thanks,